extensionsGallery.json

```diff
@@ -1534,6 +1534,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "39",
+					"extensionName": "azuredatastudio-managed-instancee",
+					"displayName": "Managed Instance dashboard",
+					"shortDescription": "Display Managed Instance properties, state, and health information",
+					"publisher": {
+						"displayName": "Jovan Popovic",
+						"publisherId": "jocapc",
+						"publisherName": "jocapc"
+					},
+					"versions": [
+						{
+							"version": "0.1.1",
+							"lastUpdated": "4/30/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",

```

Comment:
We should have `AzureDataStudio` instead of `SQLOps` or `VisualStudio.Code` in assetTypes